.gitignore

```diff
@@ -1,3 +1,167 @@
-*.pyc
-*.pyo
-recipeCache/
+docs/source
+
+# From https://raw.githubusercontent.com/github/gitignore/main/Python.gitignore
+
+ Byte-compiled / optimized / DLL files
```

Comment:
I think this is missing a # 

Also probably some parts of this file are not needed.